elements/pl-dataframe/pl-dataframe.py

```diff
@@ -0,0 +1,110 @@
+from typing import cast
+
+import chevron
+import lxml.html
+import pandas
+import prairielearn as pl
+
+NO_HIGHLIGHT_DEFAULT = False
+PREFIX_DEFAULT = ""
+SUFFIX_DEFAULT = ""
+TEXT_DEFAULT = False
+SHOW_HEADER_DEFAULT = True
+SHOW_INDEX_DEFAULT = True
+SHOW_DIMENSIONS_DEFAULT = True
+SHOW_DATATYPE_DEFAULT = False
+ADD_LINE_BREAKS_DEFAULT = False
+INTERACTIVE_DEFAULT = True
+NUM_DIGITS_DEFAULT = None
+SHOW_CODE_DEFAULT = True
+
+
+def prepare(element_html: str, data: pl.QuestionData) -> None:
+    element = lxml.html.fragment_fromstring(element_html)
+    pl.check_attribs(
+        element,
+        required_attribs=["params-name"],
+        optional_attribs=[
+            "show-index",
+            "show-header",
+            "show-dimensions",
+            "show-dtype",
+            "show-code",
```

Comment:
Let's keep in line with the `pl-variable-output` options. So, instead of `show-code`, let's use:

```suggestion
            "show-python",
```

So, in the _odd_ case we want to support MATLAB/Mathematica/R we could add the language tabs in.

Reply:
That makes sense, best to use similar attributes to other elements.